packages/x-tailwindcss/src/helpers/theme.ts

```diff
@@ -102,23 +102,29 @@ export default {
     large: '2800px'
   },
   spacing: {
-    'spacing-1': 1,
-    'spacing-2': 2,
-    'spacing-4': 4,
-    'spacing-8': 8,
-    'spacing-12': 12,
-    'spacing-16': 16,
-    'spacing-20': 20,
-    'spacing-24': 24,
-    'spacing-32': 32,
-    'spacing-40': 40,
-    'spacing-48': 48,
-    'spacing-56': 56,
-    'spacing-64': 64,
-    'spacing-80': 80,
-    'spacing-96': 96,
-    'spacing-128': 128,
-    'spacing-152': 152
+    1: 1,
+    2: 2,
+    4: 4,
+    8: 8,
+    12: 12,
+    16: 16,
+    20: 20,
+    24: 24,
+    32: 32,
+    40: 40,
+    48: 48,
+    56: 56,
+    64: 64,
+    80: 80,
+    96: 96,
+    112: 112,
+    128: 128,
+    144: 144,
+    152: 152,
+    176: 176,
+    208: 208,
+    240: 240,
+    272: 272
```

Comment:
why are there more sizes than defined here?
https://www.figma.com/file/9a1DMgqIHt5ZZBvS2waaOs/ARCHETYPE-DS?node-id=829%3A10321

Reply:
🤷‍♂️ beats me. I inherited this part of the code, will stick to the sizes in the figma.